### DISCUSSION TITLE
Move asdf-venom to asdf-community

### DISCUSSION THREAD

**Discussion by aabouzaid** [asdf-community/.github#46] [General]
Hi :wave: 

I'd like to move my project [asdf-venom](https://github.com/aabouzaid/asdf-venom) to [asdf-community](https://github.com/asdf-community/).

The plugin is based on [asdf-plugin-template](https://github.com/asdf-vm/asdf-plugin-template).

Best :rocket:

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
